src/lib.rs

```diff
@@ -1626,3 +1626,66 @@ impl Timecode {
         }
     }
 }
+
+pub struct APIInformation {
+    implementation: *mut IDeckLinkAPIInformation,
+}
+
+unsafe impl Send for APIInformation {}
+
+impl APIInformation {
```

Comment:
I see you added a binding, but there's still no public way to create one of these right?

```suggestion
impl APIInformation {
    pub fn new() -> Self {
        unsafe {
            Self{
                implementation: create_decklink_api_information_instance(),
            }
        }
    }
```

Reply:
Absolutely right! I am sure I will get this problem when using `APIInformation` in `avs-agent`.  Should it return a `Result` type in case `create_decklink_api_information_instance` fails?

Reply:
Yeah, probably. Or maybe an Option would make more sense since we wouldn't have an error code if it fails.

Reply:
okay